lib/app/widgets/app_bar.dart

```diff
@@ -3,6 +3,10 @@ import 'package:flutter/material.dart';
 import 'menu.dart';
 
 class MyAppBar extends StatefulWidget {
+  final List<Widget> actions;
+
+  MyAppBar({this.actions = const []});
```

Comment:
The list is spreaded below, so an assertion to make sure it's not null may make sense here.